packages/web/src/components/spa/HackAUBG/HackAUBG.jsx

```diff
@@ -31,42 +30,61 @@ export const HackAUBG = () => {
-                        anchorList, // list of anchors
-                        false, // hackAUBG button
-                        'rgba(0,0,0,.5)', // desktop background color nav
-                        true, // sticky desktop nav
-                        '#222222', // mobile nav background color when not opened (default transparent)
-                        'gray', // mobile background color nav when opened
-                        '#e2d7fc', // anchor color
-                        'white', // desktop anchor hover color
-                        'dark gray', // mobile anchor hover color
-                    )
-                }
-            />
-            <LandingPage />
-            <AboutHackathon />
-            <JourneySection />
-            <RegistrationForm />
-            <MentorsSection />
-            <JudgesSection />
-            <VideoSection />
-            <ScheduleHackathon />
-            <GradingCriteria />
-            <AwardsSection />
-            <Sponsors />
-            <FaqSection />
-            <Footer
-                color={'rgba(220,193,255,255)'}
-                iconColor={'rgb(0, 0, 0)'}
-                iconBgColor={'rgba(0, 0, 0, 0)'}
-                textColor={'#000'}
-                iconSize={'2.6em'}
-            />
+        <div className={`hackaubg-container ${showVerification ? 'showVerification' : ''}`}>
+            <div className={showVerification ? 'blur' : ''}>
+                <NavBar
+                    props={
+                        new Props(
+                            anchorList, // list of anchors
+                            false, // hackAUBG button
+                            'rgba(0,0,0,.5)', // desktop background color nav
+                            true, // sticky desktop nav
+                            '#222222', // mobile nav background color when not opened (default transparent)
+                            'gray', // mobile background color nav when opened
+                            '#e2d7fc', // anchor color
+                            'white', // desktop anchor hover color
+                            'dark gray', // mobile anchor hover color
+                        )
+                    }
+                />
+                <LandingPage />
+                <AboutHackathon />
+                <JourneySection />
+                <RegistrationForm />
+                <MentorsSection />
+                <JudgesSection />
+                <VideoSection />
+                <ScheduleHackathon />
+                <GradingCriteria />
+                <AwardsSection />
+                <Sponsors />
+                <FaqSection />
+                <Footer
+                    color={'rgba(220,193,255,255)'}
+                    iconColor={'rgb(0, 0, 0)'}
+                    iconBgColor={'rgba(0, 0, 0, 0)'}
+                    textColor={'#000'}
+                    iconSize={'2.6em'}
+                />
+            </div>
+            {showVerification && (
```

Comment:
Can't you just pass this className to the div inside the verifyaccount component?